tests/MongoDB.Driver.Tests/Specifications/Runner/MongoClientJsonDrivenTestRunnerBase.cs

```diff
@@ -195,6 +195,12 @@ protected virtual void CreateCollection(IMongoClient client, string databaseName
             database.CreateCollection(collectionName);
         }
 
+        protected virtual IDisposable ConfigureTestClientRelatedDisposable()
```

Comment:
I don't think what you are trying to do with this method works (see other comments) and if you want to keep it the name needs improving.